.github/workflows/ci-cpp.yaml

```diff
@@ -70,9 +70,9 @@ jobs:
           docker rmi $(docker images -q) -f
           df -h
 
-      - name: build pacakge
+      - name: build package
         if: steps.docs.outputs.changed_only == 'no'
-        run: mvn -B -Dorg.slf4j.simpleLogger.defaultLogLevel=ERROR package -DskipTests
+        run: mvn -B -ntp -Dorg.slf4j.simpleLogger.defaultLogLevel=ERROR -ntp install -Pcore-modules -DskipTests
```

Comment:
I think all the jobs that are running broker tests can also use just core-modules

Reply:
I mean: everything but integration tests

Reply:
not really we still have to run all the unit tests in these modules

Reply:
Yes, but unit tests are already included in broker module, there's no need to compile the entire project, just the core modules

Reply:
but no the units for io modules kafka-adaptor , slink , storm etc. I will double check after this pr which jobs need changes.